util/dump_index.py

```diff
@@ -0,0 +1,73 @@
+"""Dump the current elasticsearch index to a JSON file,
+
+This should be used to re-create the cached 1000 genomes
+example index if the indexer changes. It dumps the main
+index and its mappings, as well as the '_fields' index. 
+This tool can also be used to dump any other index.
+
+To run:
+
+virtualenv ~/virtualenv/elasticsearch
```

Comment:
opt nit -- To me, a virtualenv named `elasticsearch` only has `elasticsearch` installed. I changed the name (to dump_index) because there are other modules besides `elasticsearch`.